setup.py

```diff
@@ -21,7 +21,7 @@
         "dataclasses;python_version<'3.7'",
         "marshmallow>=3.0.0",
         "microcosm>=2.12.0",
-        "microcosm-caching>=0.1.0",
+        "microcosm-caching>=0.2.0",
```

Comment:
@clatko-globe  FYI bumping this to make sure we include support for `ttl` parameter, which I added after the `0.1.0` release

Reply:
cool. thanks for the comments.